src/gbd_mapping/risk_factor.py

```diff
@@ -1871,9 +1872,9 @@
             yll_only=False,
             yld_only=False,
             yll_age_group_id_start=3.0,
-            yll_age_group_id_end=389.0,
+            yll_age_group_id_end=388.0,
```

Comment:
Shouldn't these IDs be ints?

Reply:
Risk restrictions get converted to floats by gbd_mapping for some reason - even though the cause restrictions remain ints. I could coerce them but they seem to behave the same.

Reply:
Take a quick look to see if you can figure out why. If not, file a ticket for it.

Reply:
So they don't actually get converted - they are floats in the metadata. Should I keep them as is?